cla_public/static-templates/errors/50x.html

```diff
@@ -80,14 +80,14 @@
       </script>
       <div id="skiplink-container">
         <div>
-          <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+          <a href="#main-content" class="govuk-skip-link">{% trans %}Skip to main content{% endtrans %}</a>
         </div>
       </div>
 
       <div id="global-cookie-message">
         <div class="outer-block">
           <div class="inner-block">
-            <p>GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a></p>
+            <p>{% trans %}GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies{% endtrans %}</a></p>
```

Comment:
Can we perhaps split this up?
`{% trans %}GOV.UK uses cookies to make the site simpler.{% endtrand %} <a href="https://www.gov.uk/help/cookies">{% trans %}Find out more about cookies{% endtrans %}</a>`
That way we don't need the tag in the translation file.